packages/react/package.json

```diff
@@ -18,21 +18,27 @@
     "report": "WITH_REPORT=true pnpm build"
   },
   "dependencies": {
-    "@logto/browser": "^0.1.0"
+    "@logto/browser": "^0.1.0",
+    "@silverhand/essentials": "^1.1.6",
+    "react": "^17.0.2"
```

Comment:
react should be present in dev & peer deps

Reply:
done